contracts/multisig/src/verifier_set.rs

```diff
@@ -49,6 +49,9 @@ impl VerifierSet {
     pub fn hash(&self) -> Hash {
         let mut hasher = Keccak256::new();
 
+        // Length prefix the bytes to be hashed to prevent hash collisions
+        hasher.update(self.signers.len().to_be_bytes());
```

Comment:
Looks like this affects the verifier set id, which affects the multisig/prover contract